ross/bearing_seal_element.py

```diff
@@ -1736,17 +1739,17 @@ def load(file_name=""):
         return bearing_elements
 
     def dof_mapping(self):
-        return dict(x_0=0, y_0=1, z_0=2)
+        return dict(u_0=0, v_0=1, w_0=2)
 
     def dof_global_index(self):
         """Get the global index for an element specific degree of freedom for the 6DoF element."""
         global_index = super().dof_global_index()
 
         if self.n_link is not None:
             global_index = global_index._asdict()
-            global_index[f"x_{self.n_link}"] = 6 * self.n_link
-            global_index[f"y_{self.n_link}"] = 6 * self.n_link + 1
-            global_index[f"z_{self.n_link}"] = 6 * self.n_link + 2
+            global_index[f"u_{self.n_link}"] = 6 * self.n_link
+            global_index[f"v_{self.n_link}"] = 6 * self.n_link + 1
+            global_index[f"w_{self.n_link}"] = 6 * self.n_link + 2
             dof_tuple = namedtuple("GlobalIndex", global_index)
             global_index = dof_tuple(**global_index)
```

Comment:
I believe we can remove this method, since the global index are calculated in `Rotor` class.
Instead, add the attribute `self.dof_global_index = None`.

Reply:
Hello Rodrigo, good morning.

We did try to make it by inheriting this method, but due to the different naming it failed. Since it is only a "repeated" code with some modifications, I believe that it is not detrimental to the program.

Reply:
Hello, Vergílio!

You can delete both actually. the `.dof_global_index()` method from 6dof and from 4dof. They're not being called anymore on `Rotor` class. I think we forgot to delete it when we changed the way we build the global indexes.

Reply:
Oh, alright then. We will check and delete it.